automated-testing/synthetic-monitoring-tests/readme.md

```diff
@@ -0,0 +1,82 @@
+
+For such systems, the ambition of service engineering teams is to reduce to a minimum the time it takes to fix errors, or the [MTTR - Mean Time To Repair](https://en.wikipedia.org/wiki/Mean_time_to_repair). It is a continuous effort, performed on the live/production system.
+
+Synthetic Monitoring tests are a subset of tests that run in production, sometimes named Test-in-Production or Shift-Right tests. Shift right compliments and add on top of the Shift-Left paradigms that are so popular, providing modern engineering teams a broader set of tools to assure high SLAs over time.
+
+## Synthetic Monitoring tests Design Blocks [The What]
+
+Testing using synthetic data and real testing accounts injects user behaviors to the system and validates their effect, usually by passively relying on existing monitoring and alerting capabilities.
+Components of synthetic monitoring tests include **Probes**, test code/ accounts which generates data, and **Monitoring tools** placed to validate both the system's behavior under test and the health of the probes themselves.
+
+![E2E Testing Pyramid](./images/syntheticMonitoring.png)
+
+### Probes
+
+A Synthetic Monitoring test is, in fact, very related to black-box tests and would usually focus on end-to-end scenarios from a user's perspective.
+It is not uncommon for the same code for e2e or integration tests to be used to implement the probe. Probes target front-end or publicly-facing APIs.
+
+### Monitoring
+
+Given that Synthetic Monitoring tests are continuously running, at intervals, in a production environment, the assertion of system behavior through analysis relies on existing monitoring pillars used in live system (Logging, Metrics, Distributed Tracing).
+There would usually be a finite set of tests, and key metrics that are used to build monitors and alerts to assert against the known [SLO](https://en.wikipedia.org/wiki/Service-level_objective), and verify that the [OKR](https://en.wikipedia.org/wiki/OKR) for that system are maintained. The monitoring tools are effectively capturing both RUMs and synthetic data generated by the probes.
+
+## Applying Synthetic Monitoring Tests [the how]
+
+### Asserting the system under tests
+
+Synthetic monitoring tests are usually statistical. Test metrics are compared against some historical or running average with a time dimension *(Example: Over the last 30 days, for this time of day, the mean average response time is 250ms for AddToCart operation with a standard deviation from the mean of +/- 32ms)*. So if an observed measurement is within a [deviation of the norm](https://en.wikipedia.org/wiki/Standard_deviation) at any time, the services are probably healthy.
+
```

Comment:
The preceding paragraphs seem to suggest a kind of outline for how to go about building synthetic test cases: 
step 1: determine the metric to be validated (functional result, latency, etc.)
step 2: build a piece of automation that measures that metric against the system, and gathers telemetry into the system's existing monitoring infrastructure
Step 3: set up monitoring alarms/actions/responses that detect the failure of the system to meet the desired goal of the metric
Step 4: run the test case automation continuously at an appropriate interval.

or something along those lines.  IMO, it would be helpful to have that step-by-step (at a high level -- you already have links to resources for the details) to inform the reader of roughly what this class of tests entails.